windows/src/Makefile

```diff
@@ -202,21 +192,35 @@ build-release-wrapper-oem:
     $(MAKE) setup
 !else
     echo Not building OEM releases
-!endif    
+!endif
 
 #
 # Tools needed to complete a build
 #
 
 global-versions: mkver-app
+    # Build our root version.txt
     cd $(ROOT)\src
+!ifdef BUILD_NUMBER
+    # If CI has defined a version for us, then use that.
+    $(MKVER_APP) -c $(ROOT)\src\version.txt -b $(BUILD_NUMBER)
+!else
+    # Otherwise, we use the major version from resources/version.md and build 999 for 'debug'
+    $(MKVER_APP) -c $(ROOT)\src\version.txt -r $(KEYMAN_ROOT)\resources\version.md
```

Comment:
Windows doesn't care about casing, but I note the file is `VERSION.md`.

Too bad we're not consistent with the `.md` and `.MD` files across the repos :/

Reply:
Good catch... Consistency in casing is such a hassle tho' :)

Reply:
```
$ find -name "*.MD"
./windows/src/global/inst/data/keyboard_info/readme.MD
./windows/src/test/integration-tests/readme.MD
./windows/src/test/manual-tests/installer/readme.MD
./windows/src/test/manual-tests/readme.MD
./windows/src/test/unit-tests/readme.MD
```

I will fix those five and then I think we are getting reasonably consistent with extension case. There are more lower case base names -- the most egregious being `history.md` so I won't try and fix all those instances at this time (especially don't want to touch `history.md`...).